mozilla/fx-private-relay
# Fix styling source maps by importing react-toastify styling through scss file

This PR fixes #2745.

The source map problems were being caused by the react-toastify styling, despite the css file having a map file. Swapping the css file with a scss file stops the warnings on console and makes the source map viewable on the developer tools style editor.

The React-Toastify website (https://fkhadra.github.io/react-toastify/how-to-style/) recommends using the scss files as one of the ways to style the component, so there is an equal risk in breaking changes during a package update as before. Importing scss files from modules already happens with `@mozilla-protocol/core`, so I assume there aren't any objections against using scss files over css files.

Before

![image](https://user-images.githubusercontent.com/16689477/233777701-f563ba26-a0e2-4b40-97ae-52a4033b83cc.png)

After

![image](https://user-images.githubusercontent.com/16689477/233777463-663409e9-e36f-4d61-8047-6d050eadd0a5.png)

How to test:

1. Open Relay website.
2. Inspect any React-Toastify style file on developer tools.
3. Inspect console for any source map related warnings.
